x-pack/test/alerting_api_integration/security_and_spaces/group1/tests/alerting/backfill/get.ts

```diff
@@ -123,7 +120,11 @@ export default function getBackfillTests({ getService }: FtrProviderContext) {
                 start: '2023-10-19T12:00:00.000Z',
                 end: '2023-10-25T12:00:00.000Z',
               },
-              { rule_id: ruleId2, start: '2023-10-19T12:00:00.000Z' },
+              {
```

Comment:
Increasing the duration of the scheduled backfill so it doesn't finish and get deleted before the get tests finish.